### PR TITLE
Update logos on resources page to new stacked ones

### PR DESCRIPTION
**Pull request by bartaz**
## Done

Updated logos on resources page with new stacked ones.

Fixes WD-3039

## QA

- Go to resources page: https://design-ubuntu-com-287.demos.haus/resources
  - make sure logos are updated to new versions
 
logos

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
why are we showing the stacked ones? I think we should show the ones we use on websites, which are the linear versions where Canonical is on the same line, and in the same size as the product name

**Comment by ClementChaumel** [comment]
Are we sure the files are the right ones? They look just a bit blurred to me even though they're supposed to be SVGs 
image

**Comment by bartaz** [comment]
@ClementChaumel I switched to use SVGs directly, is that better? (they are still quite small, so may depend on your screen pixel size…)

**Comment by ClementChaumel** [comment]
> @ClementChaumel I switched to use SVGs directly, is that better? (they are still quite small, so may depend on your screen pixel size…)

Way better!

File: templates/_layouts/_root.html [hunk 1]
```diff
@@ -23,6 +23,11 @@
   'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
   })(window,document,'script','dataLayer','GTM-NV2KV6F');</script>
   <!-- End Google Tag Manager -->
+
+  <style>
+    /* remove rounded corners from images to avoid distorting logos */
+    img { border-radius: 0; }
+  </style>
```
Comment (ClementChaumel) [inline templates/_layouts/_root.html:30]: Shouldn't this be in `main.scss`? Feels weird to have inline styles

**Review by ClementChaumel — APPROVED** [review]
LGTM! (with a comment)

**Comment by bartaz** [comment]
@lyubomir-popov I updated the logos with latest versions from Marcus, and increased size to 3rem to make the small "Canonical" text readable.

Let me know if it's fine as is.

**Comment by lyubomir-popov** [comment]
@bartaz why so few? My understanding was that we would put all publicly available logos

**Pull request merged**